code/game/objects/structures/artstuff.dm

```diff
@@ -397,3 +426,18 @@
 	log_admin("[key_name(user)] has deleted a persistent painting made by [author].")
 	message_admins("<span class='notice'>[key_name_admin(user)] has deleted persistent painting made by [author].</span>")
 
+
+// Until somebody bigger brained wants to mess around with painting construction.
+/obj/structure/sign/painting/screwdriver_act(mob/user, obj/item/I)
+	return
+
+/obj/structure/sign/painting/proc/trigger_alarm()
+	set waitfor = FALSE
+	if(alert && is_station_contact(z))
+		var/area/alarmed = get_area(src)
+		alarmed.burglaralert(src)
+		visible_message("<span class='danger'>The burglar alarm goes off!</span>")
+		// Play the burglar alarm three times
+		for(var/i = 0, i < 4, i++)
+			playsound(src, 'sound/machines/burglar_alarm.ogg', 50, 0)
+			sleep(74) // 7.4 seconds long
```

Comment:
I'm not sure this sleep is necessary, as there isn't any code that gets executed after it.

Reply:
It is, because it loops 7.4 second each time, then restarts the for loop.

Reply:
Yeah nevermind, I somehow missed the for loop despite commenting on it already :clap: